specs/waku/v2/waku-v2.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Waku
-version: 2.0.0-beta1
+version: 2.0.0-beta2
```

Comment:
Not sure we need to bump this in PR, though it might be a good idea to update the reference to store protocol here to beta2

Honestly the versioning of main spec isn't obvious right now, so we might need to figure out what the best flow looks like here. It doesn't immediately correspond to a protocol string, and other specs will move out of sync, etc. For now I'd leave it be and we can bump this separately.